zerver/openapi/zulip.yaml

```diff
@@ -6672,6 +6672,135 @@ paths:
                     description: |
                       An example JSON error response when attempting to deactivate the only
                       organization owner in an organization:
+  /users/me/alert_words:
+    get:
+      operationId: get-alert-words
+      summary: Get all alert words
+      tags: ["users"]
+      description: |
+        List all of the requesting user's configured [alert words][alert-words].
+
+        [alert words]: /help/pm-mention-alert-notifications#alert-words
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - $ref: "#/components/schemas/SuccessDescription"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      alert_words:
+                        type: array
+                        description: |
+                          An array of strings where each string is an alert word specified by
+                          the requesting user.
+                        items:
+                          type: string
+                    example:
+                      {
+                        "result": "success",
+                        "msg": "",
+                        "alert_words": ["natural", "illustrious"],
+                      }
+    post:
+      operationId: add-alert-words
+      summary: Add alert words
+      tags: ["users"]
+      description: |
+        Add words to the requesting user's set of [alert words][alert-words].
+
+        [alert-words]: /help/pm-mention-alert-notifications#alert-words
+      parameters:
+        - name: alert_words
+          in: query
+          description: |
+            An array of strings to be added to the requesting user's set of alert words.
```

Comment:
Do we want to note the string length limitation here? And then also maybe add the error response ...

```console
{
   "code" : "BAD_REQUEST",
   "msg" : "alert_words[0] is too long (limit: 100 characters)",
   "result" : "error"
}
```

Reply:
Added the error response. I did not find any character limit being documented for similar parameters in the documentation.